Home_work22/InplaceArray.h

```diff
@@ -0,0 +1,8 @@
+#pragma once
+#include "DynamicArray.h"
+template <typename T, const std::size_t N> class InplaceArray: public DynamicArray<T>  {
```

Comment:
It should combine best of two worlds: on-stack memory usage for small amount of elements and on-heap memory allocation for large amount of elements